packages/document-picker/react-native-document-picker.podspec

```
@@ -21,8 +21,7 @@ Pod::Spec.new do |s|
     'DEFINES_MODULE' => 'YES',
   }
 
-  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
-    # RN 71+
+  if respond_to?(:install_modules_dependencies, true)
```

```suggestion
  if ENV['RCT_NEW_ARCH_ENABLED'] == '1' then
```

this should have no effect on the fix. I'm going to make some more adjustments to this later

---

Okay I agree. Thank you!